source/installation/install-on-an-unattended-server/rhel-centos-server-installation.txt

```diff
@@ -9,16 +9,148 @@ RHEL/CentOS Systems Installation
-on installation and configuration, see the installation tasks. 
+You can install Relational Migrator on a RHEL/CentOS server. This installation 
+method allows Relational Migrator to run on an unattended server. By running on an 
+unattended server, Relational Migrator binds to any IP and port on the server 
+and exposes it as a web application.
 
-Get Started
------------
+About this Task
+---------------
 
-- :ref:`Install on RHEL/CentOS <install-rhel-server>`
+Relational Migrator uses the following file locations on RHEL/CentOS:
 
-- :ref:`Run Relational Migrator as a System Service on RHEL/CentOS <rhel-system-service>`
+.. include:: /includes/fact-file-location-linux.rst
 
+
+Steps
+-----
+
+.. _install-rhel-server:
+
+Install Relational Migrator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Download the latest ``.rpm`` binary from the `release page <https://www.mongodb.com/try/download/relational-migrator>`__.
+
+#. Install Relational Migrator.
+
+   .. code-block:: bash
+
+      sudo yum install mongodb-relational-migrator-<version_number>-1.x86_64.rpm
+
+#. (Optional) To connect to Oracle or MySQL drivers, download and place the drivers 
+   into your Relational Migrator directory. 
+
+   a. To download and install MySQL or Oracle drivers, see the following:
+
+      - `MySQL: latest 8.3.x version, Platform Independent Version <https://dev.mysql.com/downloads/connector/j/>`_
+
+      - `Oracle: 21.6.0.0 of ojdbc11.jar from the Oracle 21c <https://www.oracle.com/database/technologies/appdev/jdbc-downloads.html>`_
+
+   b. Copy the driver files to ``/opt/mongodb-relational-migrator/lib/app/lib``.
+
+      To copy the driver, use the following example:
+
+      .. code-block:: bash
+        
+         curl https://download.oracle.com/otn-pub/otn_software/jdbc/216/ojdbc11.jar -O -L -b -o
+
+   For more information, see :ref:`Install on a Local Machine <local-machine-install>`.
+
+#. Change the directory to the Relational Migrator ``/bin`` location and start the application.
+
+   .. code-block:: bash
+
+      cd /opt/mongodb-relational-migrator/bin
+      
+      ./mongodb-relational-migrator
+
+#. Enable Relational Migrator to run as an unattended standalone server.
+
+   In your ``user.properties`` file, uncomment ``spring.profiles.active: unattended``. 
+
+#. (Optional) Update TLS/SSL configuration properties.
+
+   In your ``user.properties`` file, uncomment and update TLS/SSL configuration 
+   properties.
+
+   It is recommended that the unattended server installation is supplemented with 
+   TLS/SSL configuration because Relational Migrator can be bound to any IP and port on 
+   the server. For additional information, see `SSL Configuration Properties <https://docs.spring.io/spring-cloud-skipper/docs/2.8.4/reference/htmlsingle/#configuration-security-enabling-https>`__.
+
+#. (Optional) Update the port that Relational Migrator runs on.
+
+   By default, Relational Migrator is configured to run on port 8278.
```

Comment:
Similar comment as above re backticks for the port number. This applies throughout.